app/models/game.rb

```diff
@@ -21,8 +21,8 @@ def populate_board!
         # 1  r  k  b  q  K  b  k  r     white side
 
         for i in 1..8 do
-            chess_pieces.create(x_pos: i, y_pos: 2, color: 'white', type: 'Pawn')
-            chess_pieces.create(x_pos: i, y_pos: 7, color: 'black', type: 'Pawn')
+            chess_pieces.create(x_pos: 2, y_pos: i, color: 'white', type: 'Pawn')
```

Comment:
this one matches the rest of the chesspiece. 👍 sorry for leaving so many comments. I just figured out how to comment on a specific line. 😂